intern/models.py

```diff
@@ -58,8 +55,15 @@ class InternRole(models.Model):
     semester_start = models.ForeignKey(Semester, related_name='start', default=get_semester)
     semester_end = models.ForeignKey(Semester, related_name='end', null=True, blank=True)
     comments = models.CharField(max_length=300, null=True, blank=True)
-    date_added =models.DateField(default=timezone.now)
+
+    date_added = models.DateField(default=timezone.now)
```

Comment:
Might want to use `DateField(auto_now_add=True)` as we are using elsewhere
See https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.DateField.auto_now_add